terraform/vars/terraform-dev.tfvars

```diff
@@ -5,6 +5,7 @@ lambdas_path = "../src/lambdas"
 policies_path = "../terraform/modules/datapump/policies"
 lambda_layers_path = "../src"
 
-data_api_uri = "http://gfw-data-api-elb-feature-MultiCo-724307654.us-east-1.elb.amazonaws.com"
+//data_api_uri = "https://staging-data-api.globalforestwatch.org"
+data_api_uri = "http://gfw-data-api-elb-16bpp-pngs-727012843.us-east-1.elb.amazonaws.com"
```

Comment:
Since the dev workspaces are not persistent over time it is probably best to build against staging (unless of cause there is a feature only in dev which you depend on)